src/kakarot/library.cairo

```diff
@@ -225,6 +225,16 @@ namespace Kakarot {
         return (starknet_contract_address=starknet_contract_address);
     }
 
+    // @notice Set the account implementation class hash
+    // @param account_contract_class_hash The new account implementation class hash
+    func set_account_contract_class_hash{
+        syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
+    }(account_contract_class_hash: felt) {
+        Ownable.assert_only_owner();
```

Comment:
I know that we have this everywhere, but the `assert_only_owner` and other similar restriction should be put in the contract and not in the library.
I've created an issue to fix them all https://github.com/kkrt-labs/kakarot/issues/1094

Reply:
done